app/(main)/league/[leagueId]/entry/all/page.tsx

```diff
@@ -87,19 +116,45 @@ const Entry = ({
     if (!user.id || user.id === '') {
       return;
     }
-
     getCurrentGameWeek();
     getAllEntries();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [user]);
 
   return (
     <>
       {loadingData ? (
         <GlobalSpinner />
       ) : (
-        <>
+        <div className="mx-auto max-w-3xl pt-10">
```

Comment:
Why can't we just apply these classes to header here and remove div?

Reply:
> Why can't we just apply these classes to header here and remove div?

These classes are not for the header but for the entire section with content on the page.